pyfesom2/datasets.py

```diff
@@ -141,20 +141,31 @@ def fesom_mesh_to_xr(path: str, alpha: int = 0, beta: int = 0, gamma: int = 0) -
     mesh = load_mesh(path, abg=[alpha, beta, gamma])
     ncyclic_inds = get_no_cyclic(mesh, mesh.elem)
     triangles = mesh.elem[ncyclic_inds]
+    if any(mesh.zlev < 0.):
```

Comment:
Would that be easier just to apply `numpy.absolute` without if ?

Reply:
Thanks, right, not sure what i thought before; may be was trying to avoid one additional import line in the module... but this is better.